src/keys.c

```diff
@@ -69,6 +69,7 @@ static struct keydef_s keydef[NBKEYS] = {
 	{ "generic-copy", "c", gettext_noop("Copy") },
 	{ "generic-paste", "p ^V", gettext_noop("Paste") },
 	{ "generic-change-view", "TAB", gettext_noop("Chg Win") },
+	{ "generic-change-view-p", "KEY_BTAB", gettext_noop("Prev Win") },
```

Comment:
Is there a particular reason this is named `generic-change-view-p` (vs. `generic-change-view-prev`)? Is there a length limitation somewhere?

Reply:
There is. Under the config menu, going to the keys configuration, having it be `generic-change-view-prev` seems to overflow over the `=`, looking like
```
generic-change-view-pr=vKEY_BTAB
```

Reply:
`generic-prev-view`?

Reply:
Sounds good to me, might also be a good idea in that case to change all the `generic-change-view`to `generic-next-view` for consistency.

Reply:
Thanks! The problem is the label is used in the config file too. Changing an existing label will cause an error when loading an existing key file unless a workaround is added.

Reply:
I see, seems like it's getting out of scope of what the original pull request was about. I'll revert that for now and make an maybe issue on that so that it could be looked at in due time.

I wouldn't really know what else would need to be changed as simply grepping the whole source dir doesn't give any more hits for `generic-change-view`.